grid-demo/src/app/data/vm.generator.ts

```diff
@@ -47,6 +47,16 @@ const defaultQuery: VmQuery = {
   seed: 100,
 };
 
+const bytesPerGB = 1000000000;
+const bytesPerTB = 1000000000000;
+const stateOptions = [
```

Comment:
I think there should be a blank line here to separate the multi-line statement. ("feat: improve data generation" commit.)

Reply:
fixed